client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -281,6 +285,9 @@ export const StockLineForm = ({
               />
             }
           />
+          {invalidLocationAlert && (
```

Comment:
Nit: looks better if the alert is on top of the page

From:
<img width="477" height="336" alt="Screenshot 2025-08-01 at 3 03 00 PM" src="https://github.com/user-attachments/assets/f3d25daa-909c-4c57-b75c-63827345c67d" />

To: 
<img width="642" height="405" alt="Screenshot 2025-08-01 at 3 17 40 PM" src="https://github.com/user-attachments/assets/bc487b0d-c9de-4e52-be24-d36c60b62b0e" />